catalog/security/security-app/src/main/resources/features.xml

```diff
@@ -43,6 +49,7 @@
         <feature>catalog-app</feature>
         <feature>banner-markings-extractor</feature>
         <feature>security-defaultsecurityattributevalues</feature>
+        <feature>security-auditclassifiedaccess</feature>
```

Comment:
❗️ We don't want this installed by default. It shouldn't be if you just remove this line.